Packs/Base/Scripts/DBotShowClusteringModelInfo/DBotShowClusteringModelInfo.py

```diff
@@ -0,0 +1,75 @@
+from CommonServerPython import *
+
+# CONSTS
+BASE_INCIDENT_COLUMNS = ['id', 'created', 'name', 'Group name', 'Group id']
+
+
+def show_model_info(model_name):
+    success = False
+    start = datetime.now()
+    model = None
+
+    while not success and datetime.now() - start < timedelta(minutes=10):
+        res = demisto.executeCommand("getMLModel", {"modelName": model_name})
+        if is_error(res):
+            time.sleep(5)
```

Comment:
Not sure to understand the logic here. If error why not returning the error ?

Reply:
It's lazy loading - if the model does not exists it's waiting for the model to finish training, because it's run in parallel to the traning model widget

Reply:
oh it's in parralele ok